dotcom-rendering/src/components/DynamicSlowMPU.tsx

```diff
@@ -283,12 +283,12 @@ type NonMPUProps = {
 const NonMPUSlice = ({ trails, containerPalette, showAge }: NonMPUProps) => {
 	return (
 		<UL direction="row" padBottom={true}>
-			{trails.slice(0, 4).map((card, cardIndex) => {
+			{trails.slice(0, 4).map((card, cardadIndex) => {
```

Comment:
I think this can stay `cardIndex` as it's inside the `NonMPUSlice`

Reply:
Thanks! find and replace gone astray